src/Lipe/ruleset.xml

```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0"?>
+<ruleset name="Lipe">
+    <description>Custom coding standards to improve sniffers</description>
+    <!-- Exclude overridden Sniffs -->
+    <rule ref="WordPressVIPMinimum">
+        <exclude name="WordPressVIPMinimum.JS.DangerouslySetInnerHTML.Found"></exclude>
+    </rule>

```

Comment:
With this approach, we are going to continue using the `WordPressVIPMinimum.JS` that hasn't been overridden and we don't need to extend or include it in this project if doesn't need to be modified.